src/main/java/bio/terra/cli/command/Main.java

```diff
@@ -89,9 +89,12 @@ public static int runCommand(String... args) {
     // allow mixing options and parameters for all commands except the pass-through app commands.
     // this is because any options that follow the app command name should NOT be interpreted by the
     // Terra CLI, we want to pass those through to the app instead
+    //
+    // Note: One exception is the gcloud pass-through app. We want to add additional option like
```

Comment:
i don't think we need to add this comment. it's confusing to know what the comment is talking about when the gcloud line is deleted.